conf/base.config

```diff
@@ -46,7 +32,30 @@ process {
     memory = { check_max( 80.GB * task.attempt, 'memory' ) }
     time = { check_max( 8.h * task.attempt, 'time' ) }
   }
-  withName: "multiqc|get_software_versions" {
+
+  withName: makeHISATindex {
+    cpus = { check_max( 10, 'cpus' ) }
+    memory = { check_max( 200.GB * task.attempt, 'memory' ) }
+    time = { check_max( 5.h * task.attempt, 'time' ) }
+  }
+  withName: trim_galore {
+    time = { check_max( 8.h * task.attempt, 'time' ) }
+  }
+  withName: markDuplicates {
+    // Actually the -Xmx value should be kept lower,
+    // and is set through the markdup_java_options
+    cpus = { check_max( 8, 'cpus' ) }
+    memory = { check_max( 8.GB * task.attempt, 'memory' ) }
+  }
+  withLabel: salmon {
```

Comment:
Should we try to group together the `withLabel` and the `withName`?

Reply:
Next release maybe? this one is pretty huge already - I created an issue for you here: https://github.com/nf-core/rnaseq/issues/303